bin/emulator/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set(TARGET_NAME emulator_wrapper)
+set(SOURCES
+    wrapper.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/../assigner/src/utils.cpp)
+
+set(CMAKE_CXX_STANDARD 23)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+add_library(${TARGET_NAME} SHARED ${SOURCES})
+
+target_include_directories(${TARGET_NAME}
+    PRIVATE
+    ${CMAKE_CURRENT_SOURCE_DIR}
+    ${CMAKE_CURRENT_SOURCE_DIR}/../assigner/include
+    ${CMAKE_CURRENT_SOURCE_DIR}/../../libs/block_generator/include)
```

Comment:
This is unnecessary, directory will be added automatically because of linking with `zkEVMBlockGenerator`